### PR TITLE
[MIG][11.0] purchase_triple_discount Migrated.

### PR DESCRIPTION
**Pull request by krunal267**
Purchase Triple Discount
===================

Manage triple discount on purchase order lines.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please check CLA and CIs status

File: purchase_triple_discount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2017 Tecnativa - David Vidal
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Purchase Triple Discount',
+    'version': '11.0.0.0.0',
```
Comment (nikul-serpentcs) [inline purchase_triple_discount/__manifest__.py:5]: `version:'11.0.1.0.0'`

File: purchase_triple_discount/tests/test_purchase_discount.py [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+class TestPurchaseOrder(common.SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestPurchaseOrder, cls).setUpClass()
+        cls.partner = cls.env['res.partner'].create({
+            'name': 'Mr. Odoo',
+        })
+        cls.product1 = cls.env['product.product'].create({
```
Comment (nikul-serpentcs) [inline purchase_triple_discount/tests/test_purchase_discount.py:15]: here `cls.env['product.product']` and other objects use multiple time, IMO here use like a global variable.

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

**Comment by emagdalenaC2i** [comment]
@pedrobaeza there is other PR for the migration of this module #595

**Comment by pedrobaeza** [comment]
Both PRs (#595 and #611) are in red state, so no good choice for now.

**Comment by rafaelbn** [comment]
Hello @krunal267 , thanks for this contribution.

It's 6 month since this PR withouy answering, are you going to continue or if not can we close it?

Thanks in advance!

**Comment by rafaelbn** [comment]
Hello @krunal267   , thanks for this contribution.

It's 6 month since this PR withouy answering, are you going to continue or if not can we close it?

Thanks in advance!

**Comment by pedrobaeza** [comment]
Superseeded by #732

**Pull request closed without merging**